### PR TITLE
[NODE-2592] docs: update to include atlas search (3.6)

### PR DESCRIPTION
**Pull request by reggi**
https://jira.mongodb.org/browse/NODE-2592

```
docs: update to include atlas search

Add Atlas search to `$text` docs.

NODE-2592
```

### CODE REVIEW

**Comment by ccho-mongodb** [comment]
Thanks @mbroadst, I created a ticket to add this into the new Node.js docs https://jira.mongodb.org/browse/DOCSP-10565

**Comment by ccho-mongodb** [comment]
@reggi is "docs_server" the correct code? I expected that it would reflect that the referrer was Node driver docs rather than server docs. Please advise when you get the chance.

File: docs/reference/content/tutorials/text-search.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ title = "Text Search"
 
 # Text Search
 
+> [Atlas Search](https://www.mongodb.com/cloud/atlas?tck=docs_server) makes it easy to build fast, relevance-based search capabilities on top of your MongoDB data. Try it today on [MongoDB Atlas](https://www.mongodb.com/cloud/atlas?tck=docs_server), our fully managed database as a service.
```
Comment (emadum) [inline docs/reference/content/tutorials/text-search.md:13]: I don't think this is the right link, the associated DRIVERS ticket says to link to `https://docs.atlas.mongodb.com/atlas-search/`

Comment (reggi) [reply docs/reference/content/tutorials/text-search.md:13]: fixed!

Comment (ccho-mongodb) [reply docs/reference/content/tutorials/text-search.md:13]: @reggi can you make sure to push the changes to this PR? I'll update the change on the new Node.js driver docs in this PR once you've done that, thanks:
https://github.com/mongodb/docs-node/pull/112/files#diff-eb57093533a79d809482414db9cb8d3fR3-R6

File: docs/reference/content/tutorials/text-search.md [hunk 2]
```diff
@@ -10,6 +10,8 @@ title = "Text Search"
 
 # Text Search
 
+> [Atlas Search](https://docs.atlas.mongodb.com/atlas-search?tck=docs_server) makes it easy to build fast, relevance-based search capabilities on top of your MongoDB data. Try it today on [MongoDB Atlas](https://www.mongodb.com/cloud/atlas?tck=docs_server), our fully managed database as a service.
```
Comment (reggi) [inline docs/reference/content/tutorials/text-search.md:13]: Verified working:
* [ ] https://docs.atlas.mongodb.com/atlas-search?tck=docs_server
* [ ] https://www.mongodb.com/cloud/atlas?tck=docs_server

**Pull request merged**